### ISSUE TITLE
"Check for Updates Results" points to Beta Program site with no beta program info

### ISSUE THREAD

**Issue by ferdnyc** [BiglySoftware/BiglyBT#521]
The update-check results window has a message at the bottom, "Visit the [Beta Program site](http://dev.biglybt.com/) for further information..."

That link points to `http://dev.biglybt.com/`, that in turn is redirecting to `http://www.biglybt.com/`... whiiiiich doesn't have even a trace of beta-related info on it.

Is there a better target for the "Beta Program site"? Here on Github? Another development URL?

### System Details
```
Java 1.8.0_172 (64 bit)
  Oracle Corporation
/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.172-9.b11.fc28.x86_64/jre

SWT v4757, gtk/3.22.30
Linux v4.17.3-200.fc28.x86_64, amd64 (64 bit)
B1.5.0.1_B18/4 az3
```

**Comment by ms52538** [comment]
I have a post up, for Windows 10, that experiencing a similar problem. https://github.com/BiglySoftware/BiglyBT/issues/505

In Windows 10 case, there is a quick fix with about 10 steps. Not sure if something similar can be performed for Linux.

The issue *may* be related with both clients searching the wrong address for updates.

**Comment by ferdnyc** [comment]
And now that I've realized that the Beta Program sidebar module points to https://biglybt.tumblr.com/, that seems like the obvious place to link "Visit the Beta Program site for further information..." to as well. (Either  by opening the Beta Program tab in Bigly, or launching the tumblr site in the user's external browser.)

**Comment by parg** [comment]
Good idea ;)

**Comment by ferdnyc** [comment]
Fixed in 1.5.0.1_B20.
